src/main/java/com/bootcamp/demo/model/Card.java

```diff
@@ -41,6 +44,15 @@ public void setCVV(String CVV) {
         this.CVV = CVV;
     }
 
+    @JsonIgnore
```

Comment:
You could return the preferred status as well so the get card / list contains this information as well.